client/webserver/site/src/js/registry.ts

```diff
@@ -4,14 +4,14 @@ declare global {
     enableLogger: (loggerID: string, enable: boolean) => void
     recordLogger: (loggerID: string, enable: boolean) => void
     dumpLogger: (loggerID: string) => void
-    localeDiscrepancies: () => void
     testFormatFourSigFigs: () => void
     testFormatRateFullPrecision: () => void
     user: () => User
     isWebview?: () => boolean
     webkit: any | undefined
     openUrl: (url: string) => void
     sendOSNotification (title: string, body?: string): void
+    clearLocale (): void
```

Comment:
This is an unfortunate side effect of these changes. A better solution is perhaps possible. The translations cached on the front end are keyed on a `(lang, commitHash)` tuple. But for developers making changes, the `commitHash` won't change until they actually commit. Ideally, the `--reload-html` config flag would enable per-request reloading of the locale maps, but since we're caching on the front-end, we won't see any changes until the commit hash changes. So updating the translations without committing necessitates calling `clearLocale`, an obvious burden and potential point-of-confusion for developers.

Reply:
I'm just ignoring the cache in dev mode (`process.env.NODE_ENV === 'development'`). Restart of dexc still required if you add new translations though.